sdks/java/io/hcatalog/build.gradle

```diff
@@ -41,6 +41,17 @@ test {
   ignoreFailures true
 }
 
+configurations.testRuntimeClasspath {
+  resolutionStrategy {
+    def log4j_version = "2.4.1"
```

Comment:
Somehow I had to redefine this here to make it work, bad to have this repeated but it is the only workaround I could find.

Reply:
Arghh what a silly mistake, my excuses I opened #14751 to upgrade this dep to the [minimum compatible version](https://stackoverflow.com/questions/52700803/im-getting-noclassdeffounderror-org-apache-logging-log4j-util-reflectionutil) with Hive dependencies, wondering if we should also cherry-pick it for the release. Thanks for noticing @aaltay !

Reply:
It is a pity we cannot move to the latest log4j version because of Hive's HCatalog dependency constraints. The bad part is that this will still trigger security analysis tools warnings, but I have no idea of how to workaround this apart of doing a full upgrade of  Hive to the latest but that does not seem to be so simple either, for ref [BEAM-9351](https://issues.apache.org/jira/browse/BEAM-9351).

Reply:
Thank you @iemejia. I do not have an opinion about cherry picking to the release or not. If it helps, I am not aware of any user issues or bug reports. I noticed this coincidentally.